wifiphisher/common/accesspoint.py

```diff
@@ -119,9 +119,14 @@ def start_dhcp_dns(self):
                 dhcpconf.write("server=%s" % (constants.PUBLIC_DNS,))
             else:
                 dhcpconf.write("address=/#/%s" % (constants.NETWORK_GW_IP,))
-
-        subprocess.Popen(['dnsmasq', '-C', '/tmp/dhcpd.conf'],
-                         stdout=subprocess.PIPE, stderr=constants.DN)
+        # catch the exception if dnsmasq is not installed
+        try:
+            subprocess.Popen(['dnsmasq', '-C', '/tmp/dhcpd.conf'],
+                             stdout=subprocess.PIPE, stderr=constants.DN)
+        except OSError:
+            print ("[" + constants.R + "!" + constants.W + "] " +
+                   "Failed to resolve dnsmasq dependency!")
```

Comment:
Maybe change the error to "dnsmasq is not installed!"? Same for hostapd.

Reply:
sure : )